<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGroupBox</class>
 <widget class="QGroupBox" name="MainGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Zamawiający</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="surnameLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Nazwisko</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="numberLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Numer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="phoneLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Telefon</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Imię</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="countryLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Kraj</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="streetLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Ulica</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="cityLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Miasto</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="emailLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>e-mail</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="disabilityLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Niepełnosprawność</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="postalCodeLineEdit">
     <property name="minimumSize">
      <size>
       <width>147</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Kod pocztowy</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>streetLineEdit</tabstop>
  <tabstop>numberLineEdit</tabstop>
  <tabstop>postalCodeLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>countryLineEdit</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>disabilityLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
